src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incentives/IncentivesSynchronisationService.kt

```diff
@@ -40,9 +41,21 @@ class IncentivesSynchronisationService(
 
         if (!this.currentIep) {
           log.info("updating current IEP $currentIep \nfollowing update to non current IEP: ${this@run}")
-          // send message to synchronis current record
 
-          resynchroniseCurrentIncentive(iepEvent.bookingId)
+          // nomis allows an IEP creation and updates to other IEPs in the same transaction. This delay will mitigate against an update being
+          // processed before a create request (and potentially creating a duplicate in nomis)
+          queueService.sendMessage(
+            IncentiveMessages.SYNCHRONISE_CURRENT_INCENTIVE,
+            MigrationContext(
+              type = MigrationType.INCENTIVES, "dummy", 0,
+              body = IncentiveMapping(
```

Comment:
minor for a future PR I think this could be an a different object with just `nomisBookingId`